systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaAdminClient.java

```diff
@@ -0,0 +1,258 @@
+        if ((builder.topicName == null || builder.topicName.isEmpty()) && !(builder.topicOperation.equals("help") || builder.topicOperation.equals("list")))
+            throw new InvalidParameterException("Topic name (or 'prefix' if topic count > 1) is not set.");
+
+        replicationFactor = (builder.replicationFactor <= 0) ? 1 : builder.replicationFactor;
+        partitions = (builder.partitions <= 0) ? 1 : builder.partitions;
+        topicCount = (builder.topicCount <= 0) ? 1 : builder.topicCount;
+        topicOffset = builder.topicOffset;
+        adminName = builder.adminName;
+        bootstrapAddress = builder.bootstrapAddress;
+        topicName = builder.topicName;
+        additionalConfig = builder.additionalConfig;
+        namespaceName = builder.namespaceName;
+        topicOperation = builder.topicOperation;
+    }
+
+    public String getAdminName() {
+        return adminName;
+    }
+
+    public String getBootstrapAddress() {
+        return bootstrapAddress;
+    }
+
+    public String getTopicName() {
+        return topicName;
+    }
+
+    public int getTopicCount() {
+        return topicCount;
+    }
+
+    public int getTopicOffset() {
+        return topicOffset;
+    }
+
+    public String getAdditionalConfig() {
+        return additionalConfig;
+    }
+
+    public String getNamespaceName() {
+        return namespaceName;
+    }
+
+    public int getPartitions() {
+        return partitions;
+    }
+
+    public int getReplicationFactor() {
+        return replicationFactor;
+    }
+
+    public String getTopicOperation() {
+        return topicOperation;
+    }
+
+    public KafkaAdminClient.Builder newBuilder() {
+        return new KafkaAdminClient.Builder();
+    }
+
+    public KafkaAdminClient.Builder updateBuilder(KafkaAdminClient.Builder builder) {
+        return builder
+            .withAdditionalConfig(getAdditionalConfig())
+            .withBootstrapAddress(getBootstrapAddress())
+            .withTopicOperation(getTopicOperation())
+            .withTopicCount(getTopicCount())
+            .withTopicOffset(getTopicOffset())
+            .withPartitions(getPartitions())
+            .withReplicationFactor(getReplicationFactor())
+            .withAdminName(getAdminName())
+            .withTopicName(getTopicName())
+            .withNamespaceName(getNamespaceName());
+    }
+
+    public KafkaAdminClient.Builder toBuilder() {
+        return updateBuilder(newBuilder());
+    }
+
+    public JobBuilder defaultAdmin() {
+        if (namespaceName == null || namespaceName.isEmpty()) {
+            LOGGER.info("Deploying {} to namespace: {}", adminName, ResourceManager.kubeClient().getNamespace());
+            namespaceName = ResourceManager.kubeClient().getNamespace();
+        }
+
+        Map<String, String> adminLabels = new HashMap<>();
+        adminLabels.put("app", adminName);
+        adminLabels.put(Constants.KAFKA_CLIENTS_LABEL_KEY, Constants.KAFKA_CLIENTS_LABEL_VALUE);
```

Comment:
Wouldn't be better to use a different label for admin client?

Reply:
No problem for me to change. I thought it's still KAFKA_CLIENTS.. ;-)